e2e/testsuite/codec.go

```diff
@@ -0,0 +1,17 @@
+package testsuite
+
+import (
+	"github.com/cosmos/cosmos-sdk/codec"
+	banktypes "github.com/cosmos/cosmos-sdk/x/bank/types"
+	govv1beta1 "github.com/cosmos/cosmos-sdk/x/gov/types/v1beta1"
+
+	simappparams "github.com/cosmos/ibc-go/v6/testing/simapp/params"
+)
+
+func Codec() *codec.ProtoCodec {
```

Comment:
Added this helper function as I ended up needing the codec again for this test.

Reply:
Ayyye, I guess we'll just add to this as needed! Thanks 🙏

Reply:
Should we use the [`DefaultCodec`](https://github.com/strangelove-ventures/ibctest/blob/main/chain/cosmos/codec.go#L16) exposed by ibctest?

Reply:
actually it looks like this function doesn't register `govv1beta1` and is not configurable in any way.